src/styles/Login.css

```diff
@@ -0,0 +1,81 @@
+.Login {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  height: 100vh;
+}
+
+.Login-Content {
+  display: flex;
+  background: #f6f7fc;
+  height: calc(100% - 100px);
+  justify-content: center;
+  margin-top: 100px;
+}
+
+.Login-Panel {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  text-align: left;
+  background: white;
+  box-shadow: 4px 4px 10px #212121;
+  width: 500px;
+  height: 500px;
+  border-radius: 20px;
+  padding: 30px;
+  margin: 100px;
+}
+
+.Login-Header-Text {
+  font-family: "ABeeZee";
+  color: #1f1f1f;
+  font-size: 50px;
+  font-weight: normal;
+  text-align: center;
+}
+
+.Create-Account-Text {
+  text-align: center;
+  font-size: 15px;
+}
+.Link-Text {
+  margin-left: 5px;
+  color: #ff0059 !important;
```

Comment:
Are !important properties needed here? The styling seems to be working for me without those.

Reply:
I'll have a look again, they weren't working for me before

Reply:
@dder725 fixed now! thanks for that